### DISCUSSION TITLE
lora微调以后合并使用vllm输出停不下来

### DISCUSSION THREAD

**Discussion by hialuoy** [zai-org/ChatGLM3#1061] [Q&A] [answered]
lora微调以后合并使用vllm输出停不下来。 各位老师有没有遇到过啊？有什么解决办法吗

**Comment by luowei-D** [comment]
在直接使用大模型的情况下也会出现输出不会停止，进入死循环的情况，会拖慢整体大模型的响应速度

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
这个应该是没有指定special token，使用chattemplate就行了
